source/stylesheets/_banner.scss

```diff
@@ -0,0 +1,37 @@
+$_banner-background-color: #e7f1ec;
```

Comment:
A couple thoughts, and I can be further persuaded here.

1. I only plan on using these here. If they were used elsewhere, I would certainly move them out to a shared thing.
2. I'd like to keep the Bitters install untouched. Unlike typical projects, we're going to want to keep it in sync with the latest release for component styling. Maybe we should be using a gem or something to lock it by versions. That wouldn't be fun to work with though, because you can't pop open a file to reference. I might reconsider this point in a separate PR.